templates/emails/onboarding.day2.hbs

```diff
@@ -0,0 +1,21 @@
+Subject: Give visibility to your backers and sponsors
```

Comment:
This is the subject of day 7. Subject for day two is "Welcome! - some quick advice from our team."

Reply:
good catch